check-diff.sh

```diff
@@ -494,12 +494,18 @@ function run_phpunit_local {
 				cd - > /dev/null
 			fi
 
-			if [ ! -z "$ABSOLUTE_VAGRANT_PATH" ]; then
+            if [ ! -z "$ABSOLUTE_VAGRANT_PATH" ]; then
 				echo "Running phpunit in Vagrant"
-				vagrant ssh -c "cd $ABSOLUTE_VAGRANT_PATH && phpunit -c $PHPUNIT_CONFIG"
+				if [ -n "$PHPUNIT_CONFIG" ]; then
+				    vagrant ssh -c "cd $ABSOLUTE_VAGRANT_PATH && phpunit -c $( echo "$PHPUNIT_CONFIG"; )"
+				else
+				    vagrant ssh -c "cd $ABSOLUTE_VAGRANT_PATH && find . -iname phpunit.xml.dist -execdir phpunit \;"
```

Comment:
This will need to use `phpunit.xml` if it exists, and otherwise fall-back to `phpunit.xml.dist`, in each directory.